var/spack/repos/builtin/packages/soapindel/package.py

```diff
@@ -0,0 +1,57 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Soapindel(MakefilePackage):
+    """SOAPindel is focusing on calling indels from the next-generation
+       paired-end sequencing data."""
+
+    homepage = "http://soap.genomics.org.cn/soapindel.html"
+    url      = "http://soap.genomics.org.cn/down/SOAPindel_20130918_2.1.7.17.zip"
+
+    version('2.1.7.17', '317ef494173969cdc6a8244dd87d06bd')
+
+    depends_on('perl', type=('build', 'run'))
+
+    def url_for_version(self, version):
+        url = 'http://soap.genomics.org.cn/down/SOAPindel_20130918_{0}.zip'
+        return url.format(version)
```

Comment:
This probably won't work for any other version of SOAPindel. You should just use a version-specific URL.